spec/spec_helper.rb

```diff
@@ -6,7 +6,8 @@
 Mumukit::Auth.configure do |c|
   c.client_id = 'foo'
   c.client_secret = Base64.encode64 'bar'
-  c.daybreak_name = ENV['MUMUKI_DAYBREAK_NAME'] || 'test.db'
+  c.daybreak_name = 'test'
```

Comment:
The `daybreak_name` is no longer necessary, since it is passed to the `PermissionsPersistence`.